src/components/componentDiff/ComponentDiffTab.js

```diff
@@ -219,6 +221,65 @@ const ComponentVersionDiff = ({
     rightVersion: rightVersion,
   })
 
+  const HandleExportDiff = () => {
+    const addedResources = []
+    const removedResources = []
+
+    diff.components_changed?.forEach(component => {
+
+      const formatResourceName = (resource, component) => {
+        if (resource.type === 'ociImage') {
+          return `[Docker] ${component.name.split('/', 1)[0]}/${resource.name}`
+        }
+        return `${component.name.split('/', 1)[0]}/${resource.name}`
+      }
+
+      component.right.resources?.added?.forEach(resource => {
+        addedResources.push(formatResourceForYaml(component.right, resource, formatResourceName))
+      })
+
+      component.right.resources?.removed?.forEach(resource => {
+        removedResources.push(formatResourceForYaml(component.right, resource, formatResourceName))
+      })
+
+
+      component.right.resources?.changed?.forEach(change => {
```

Comment:
Since always the right component is used, you could define `const component = component.right` to increase readability.